pingcap/tidb-operator
# http-server: add deleteTidbCluster api

### What problem does this PR solve?

add delete TidbCluster api for http-server

### What is changed and how does it work?


### Code changes


### Tests



1. use CreateCluster http api to create a tidb cluster
see example/create-cluster.json

2. check tidb running success
![image](https://github.com/pingcap/tidb-operator/assets/11162926/1136a173-9635-442c-8c02-25d198ba319c)
3. call DeleteCluster http api to delete the tidb cluster
```
{
  "success": true
}
```
![image](https://github.com/pingcap/tidb-operator/assets/11162926/89c2880d-8922-425e-a062-5c8ef17c94f8)
check namespace,secret,tidbmonitor and tidbInitializers all be deleted
### Side effects


### Related changes


### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```
